package/network/config/firewall/files/firewall.config

```diff
@@ -91,6 +91,7 @@ config rule
 	list icmp_type		neighbour-solicitation
 	list icmp_type		router-advertisement
 	list icmp_type		neighbour-advertisement
+        list icmp_type		fragmentation-needed
```

Comment:
This is strange, `fragmentation-needed` is a ipv4 option, see https://github.com/openwrt/firewall4/blob/master/root/usr/share/ucode/fw4.uc#L24